.github/workflows/ci.yml

```diff
@@ -31,6 +31,7 @@ jobs:
       - uses: actions/checkout@v2
         with:
           ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
```

Comment:
I don't know that this is 100% how you'll want to solve this, but CI was refusing to check out from a fork without.